lively.morphic/text/morph.js

```diff
@@ -2430,16 +2430,24 @@ export class Text extends Morph {
 
     if (evt.isShiftDown()) {
       this.selection.lead = clickTextPos;
+      if (this.editorModeName === 'richText') this.editorPlugin.showFormattingPopUp();
```

Comment:
A little more brief, assuming no other plugin implements this method:
`this.pluginInvokeFirst('showFormattingPopUp')`